src/main/java/org/usfirst/frc4904/robot/commands/VelcroPistonRetract.java

```diff
@@ -0,0 +1,13 @@
+package org.usfirst.frc4904.robot.commands;
+
+
+import org.usfirst.frc4904.robot.RobotMap;
+
+/**
+ * This command sets the state of the associated group to released.
+ */
+public class VelcroPistonRetract extends SolenoidRelease {
```

Comment:
I think  hatchIntakeRetract makes more sense

Reply:
I disagree

Reply:
because we want the command name to accurately reflect the specific action taking